setup.py

```diff
@@ -475,9 +475,11 @@ def run(self):
           ] if ONEDAL_VERSION >= 20230100 else []
          ) + (
         ['onedal.spmd',
-         'onedal.spmd.linear_model',
-         'onedal.spmd.basic_statistics'
-         ] if build_distribute else [])),
+         'onedal.spmd.basic_statistics',
+         'onedal.spmd.linear_model'
+         ] + (['onedal.spmd.neighbors']
```

Comment:
['onedal.spmd',
         'onedal.spmd.linear_model',
         'onedal.spmd.basic_statistics',
        'onedal.spmd.neighbors'
         ] instead of  +(['onedal.spmd.neighbors'])

Reply:
I think the formatting makes the conditionals confusing, what the existing logic amounts to is:
if build_distributed:
  append(onedal.spmd, onedal.spmd.linear_model, onedal.spmd.basic_statistics]
  if onedal_version >= 20230100:
    append(onedal.spmd.neighbors)